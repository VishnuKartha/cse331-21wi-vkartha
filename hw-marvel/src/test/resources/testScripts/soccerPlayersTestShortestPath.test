# Tests that the path between Messi and Neymar is the shortest path(a direct path)

# To run this test, the soccer dataset must exist at
#   hw-marvel/src/main/resources/data/soccer.tsv

LoadGraph soccer soccer.tsv

FindPath soccer Leo_Messi Neymar

