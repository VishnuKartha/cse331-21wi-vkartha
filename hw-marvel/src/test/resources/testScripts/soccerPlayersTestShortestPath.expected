# Tests that the path between Messi and Neymar is the shortest path(a direct path)

# To run this test, the soccer dataset must exist at
# hw-marvel/src/main/resources/data/soccer.tsv

loaded graph soccer

path from Leo Messi to Neymar:
Leo Messi to Neymar via Barcelona